pkg/edit/editor.go

```diff
@@ -47,17 +50,21 @@ func NewEditor(tty cli.TTY, ev *eval.Evaler, st store.Store) *Editor {
 	}
 
 	if fuser != nil {
+		ignoreLeadingSpace := eval.NewGoFn("ignore-leading-space",
```

Comment:
This name does not correspond to an actual symbol, so put it in brackets: `<ignore-cmd-with-leading-space>`